ui/app/templates/components/dashboard/quick-actions-card.hbs

```diff
@@ -0,0 +1,59 @@
+<h3 class="title is-4">Quick actions</h3>
+
+<div class="has-bottom-margin-m">
+  <h4 class="title is-6">Secrets engine</h4>
```

Comment:
To stay consistent I think (I peeked at the list view) it's engines plural. So Secrets engines.